packages/foreman/foreman/foreman.spec

```diff
@@ -1082,6 +1084,7 @@ unlink ./%{_datadir}/%{name}/db \\
 ln -sv \`pwd\`/%{_localstatedir}/lib/%{name}/db ./%{_datadir}/%{name}/db \\
 pushd ./%%{%{name}_dir} \\
 \\
+ln -s %{nodejs_sitelib} node_modules \\
```

Comment:
Makes all the node_modules for plugins and Foreman available at plugin compile time